test/integration/test_ibm_estimator.py

```diff
@@ -0,0 +1,73 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Integration tests for Estimator primitive."""
+
+from qiskit.circuit.library import RealAmplitudes
+from qiskit.quantum_info import SparsePauliOp
+
+from qiskit_ibm_runtime import IBMEstimator
+from qiskit_ibm_runtime import EstimatorResult
+from qiskit_ibm_runtime import BaseEstimator
+
+from ..decorators import run_integration_test
+from ..ibm_test_case import IBMIntegrationTestCase
+
+class TestIntegrationIBMEstimator(IBMIntegrationTestCase):
+    """Integration tests for Estimator primitive."""
+
+    def _skip_on_legacy(self):
+        if self.dependencies.auth == "legacy":
+            self.skipTest("Not supported on legacy")
+
+    @run_integration_test
+    def test_estimator_primitive(self, service):
+        """Verify if estimator primitive returns expected results"""
+        self._skip_on_legacy()
+
+        estimator_factory = IBMEstimator(service=service)
+
+        psi1 = RealAmplitudes(num_qubits=2, reps=2)
+        psi2 = RealAmplitudes(num_qubits=2, reps=3)
+
+        H1 = SparsePauliOp.from_list([("II", 1), ("IZ", 2), ("XI", 3)])
+        H2 = SparsePauliOp.from_list([("IZ", 1)])
+        H3 = SparsePauliOp.from_list([("ZI", 1), ("ZZ", 1)])
+
+        with estimator_factory(circuits=[psi1, psi2], observables=[H1, H2, H3]) as estimator:
```

Comment:
I am not passing parameters here yet since I don't know how to serialize ParameterView.